module-telemetry/src/main/java/com/mapbox/maps/module/telemetry/MapTelemetryImpl.kt

```diff
@@ -110,11 +152,18 @@ class MapTelemetryImpl : MapTelemetry {
    * @param data performance event data
    */
   override fun onPerformanceEvent(data: Bundle?) {
-    telemetry.push(
-      MapEventFactory.buildPerformanceEvent(
-        PhoneState(appContext),
-        UUID.randomUUID().toString(), data ?: Bundle()
-      )
+    val performanceEvent = MapEventFactory.buildPerformanceEvent(
+      PhoneState(appContext),
+      UUID.randomUUID().toString(), data ?: Bundle()
     )
+    sendEvent(Gson().toJson(performanceEvent))
+  }
+
+  /**
+   * Static variables and methods.
+   */
+  private companion object {
+    private const val TAG = "MapTelemetryImpl"
+    private const val EVENTS_ACCESS_TOKEN_RESOURCE_NAME = "mapbox_events_access_token"
```

Comment:
we used it for mme vs core telemetry testing to have separate tokens for mme and core telemetry (prod for mme, staging for core telemetry)
but in case of complete switch to core telemetry I think we do not need it, we need to use the same token we used for mme

Reply:
could you clarify which token should we use now? just the access token passed from constructor?

Reply:
Access token passed from constructor.

I have some additional code for setting token in my branches but it's just to have an easy filters for was events retrieval. For production we use just access token passed from constructor.

Reply:
thanks! updated 👍